csharp_dotnetcore/13.Basic-Bot-Template/DeploymentScripts/MsbotClone/bot.recipe

```diff
@@ -0,0 +1,16 @@
+{
+  "version": "1.0",
+  "resources": [
+    {
+      "type": "endpoint",
+      "id": "http://localhost:3978/api/messages",
+      "name": "development",
+      "url": "http://localhost:3978/api/messages"
+    },
+    {
+      "type": "luis",
+      "id": "basic-bot-LUIS",
```

Comment:
This seems incorrect. this is how the basic bot on nodejs side looks like. You might have run msbot export with a version 1.0 bot file .. 

{
  "version": "1.0",
  "resources": [
    {
      "type": "endpoint",
      "id": "1",
      "name": "development",
      "url": "http://localhost:3978/api/messages"
    },
    {
      "type": "luis",
      "id": "34",
      "name": "basic-bot-LUIS"
    }
  ]
}